tests/integration/200.command.dev.test.js

```diff
@@ -37,10 +37,11 @@ testMatrix.forEach(({ args }) => {
         })
         .withRedirectsFile({
           redirects: [
-            { from: `/api/*`, to: `/.netlify/functions/echo?a=1&a=2`, status: '200' },
-            { from: `/foo`, to: `/`, status: '302' },
-            { from: `/bar`, to: `/?a=1&a=2`, status: '302' },
-            { from: `/test id=:id`, to: `/?param=:id` },
+            { from: '/api/*', to: '/.netlify/functions/echo?a=1&a=2', status: '200' },
+            { from: '/foo', to: '/', status: '302' },
+            { from: '/bar', to: '/?a=1&a=2', status: '302' },
+            { from: '/test id=:id', to: '/?param=:id' },
+            { from: '/baz/*', to: '/.netlify/functions/echo?query=:splat' },
```

Comment:
Small note here:

The redirects rules were using template literal strings instead of conventional ones so I've changed them.